cost/azure/instances_log_analitics_utilization/azure_instance_log_analytics_utilization.pt

```diff
@@ -0,0 +1,522 @@
+name "Azure Instances Utilization from Log Analytics"
+rs_pt_ver 20180301
+type "policy"
+short_description "Gathers instance utilization data from Azure Log Analytics and tags underutilized instances.  See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/instances_log_analitics_utilization) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "low"
+category "Cost"
+
```

Comment:
Need to add the permissions declaration for the cred

Reply:
Added